### PR TITLE
feat(338): add new pitchdeck slides to homepage

### LINKED ISSUE #338: Content: Add Home pg "pitch deck" introduction
The existing home doesn't introduce people to the whole concept of Clean & Green Philly. That happens on the About page. We want to change that and move the "pitch deck" for Clean & Green Philly right on the home page. 

- Here's the Figma prototype. https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=2266%3A23177&mode=design&t=jX4y1auMbn2sVoGD-1
- Add these 8 new sections to the homepage including
  - Heading
  - Body text
  - Button
  - Image
- Here's the images exported. [Home Page Images.zip](https://github.com/CodeForPhilly/vacant-lots-proj/files/14240688/Home.Page.Images.zip)

Image

### Dev Notes:

- Some headings are need to be numbered and indented. https://share.cleanshot.com/1WrldLSD I looked up if that was possible and found the start attribute. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#using_the_start_attribute
```
<h2 class="heading-2xl">
  <ol start="2">
    <li>Get data-driven suggestions on how to legally get access.</li>
  </ol>
</h2>
```

### PR DESCRIPTION
**Pull request by paulhchoi**
## Description

This PR updates the homepage based on the latest figma guidelines ([linked here](https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=2266%3A23177&mode=design&t=jX4y1auMbn2sVoGD-1)).

Leaving as draft for @thansidwell and stakeholders to review — specific feedback requests include:
1. I copied some of the graphics directly from the figma, but I believe you meant some of them as placeholders for now — would love your feedback on the finalized versions of those section graphics?
2. I assumed the final "Get Started" link went to `/map`, but that was a shot in the dark.
3. I could use more direction on the mobile view sizes for the "three blue boxes" graphic.

## Desktop (1980px)
screencapture-localhost-3000-2024-03-13-01_11_30

## Mobile (375px)
screencapture-localhost-3000-2024-03-13-01_11_48

Closes #338

### CODE REVIEW

File: src/app/components/InfoGraphicSection.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * Initially developed for use on the 'pitch deck' landing page.
+ */
+export const InfoGraphicSection = (props: InfoGraphicProps) => {
+  const { header, body, link } = props;
+
+  // Dynamically renders the graphic content based on prop type.
+  const renderGraphicContent = () => {
+    if ("image" in props) {
+      return (
+        <Image
```
Comment (paulhchoi) [inline src/app/components/InfoGraphicSection.tsx:43]: @brandonfcohen1 I had question regarding this repos usage of images: I saw that some existing images were stored in the `/public` folder, where its path was directly reference as a string from `import { Image } from "@nextui-org/react"`. What were the choices made to not use `Image from next/image;` instead?

As far as I'm aware, by using `next/image`, you get optimized image compressions and images are sized and served based on the users viewports that I don't believe you'd get by just hosting directly from the `/public` folder.

Comment (paulhchoi) [reply src/app/components/InfoGraphicSection.tsx:43]: Actually yea, I think my hunch is correct — running lighthouse on the prod homepage gives 86 for the desktop and 68 for mobile, with big penalties coming from the unoptimized images. Might be worth creating a ticket to refactor all usages of `Image from @next-ui` to `Image from next/image`

desktop
desktop

mobile
mobile

**Comment by marvieqa** [comment]
@paulhchoi I did a quick test using a screen reader to see how the list is read out, unfortunately it isn't favourable. It's confusing to have 3 lists that house only 1 item each.  I'm currently researching how to make the list appear the way we wanted to through CSS so we only have one list. 

Another thing is I recommend hiding this image from screen readers since it's just a repetition of the list item, and not adding value to the meaning of this section. cc @bacitracin 

Screenshot 2024-03-13 at 17 34 34

**Comment by thansidwell** [comment]
@paulhchoi, great work on all this. Here's a few minor issues I found. 

1. The headline and subheading don't align with the photos. Can you align them to 2/3 and 1/3 layout?  https://share.cleanshot.com/PfRHxtDN
2. On small screens lets hide these two photos. It's too many photos for such a small screen. https://share.cleanshot.com/CmQrg881
3. As the screen gets wide the gap between the text and graphics isn't centered. Can you try to center it? https://share.cleanshot.com/gfcYDnQ3
4. The buttons on the home page are different than the buttons elsewhere. https://share.cleanshot.com/tkKD8htP We have about 3 different-looking variants and I'm trying to unify them into one button style with several variations with this ticket. https://github.com/CodeForPhilly/vacant-lots-proj/issues/288  Not sure what button to use. The nav buttons in your preview look like the closest to what the design is.

File: src/app/components/InfoGraphicSection.tsx (hunk 1 above)
Comment (nlebovits) [reply src/app/components/InfoGraphicSection.tsx:52]: @paulhchoi lol the choice is 100% just me learning React on the fly + relying heavily on chatgpt + copilot. Change what you want. :)

**Comment by marvieqa** [comment]
@paulhchoi RE: https://github.com/CodeForPhilly/vacant-lots-proj/issues/338#issuecomment-1994673318
I did a quick screen reader pass to see how the nested list is announced. It doesn't make sense to have a 3 different lists that only have 1 item each, so it was confusing. 
<details>
<summary>Video Demo</summary>

https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/03be05dd-63a3-4631-b1e1-d8ca30863f6d

</details>

We can just drop the `<ol>`, and make those 3 items straight `<h3>`s. It's essentially just another heading, with number in the beginning. 

That said, let's make this image a list, because it now assists the succeeding h3 headings - denoting that it's a 3 step process.
<details>
<summary>Details</summary>

- use `<ol role="list">`, `<li role="listitem">`  because Voice Over refuses to announce the list without explicit role of list/listitem
- hide the icons from screen readers
Screenshot 2024-03-13 at 23 40 21

</details>

cc @bacitracin

**Comment by paulhchoi** [comment]
@thansidwell @marvieqa requesting a review, all feedback should've been applied — thanks for the feedback thus far. 

@thansidwell thoughts on the coloring for the main cta button?:

https://github.com/CodeForPhilly/vacant-lots-proj/assets/8061917/54664f97-4c4a-4f93-809d-225fde0716d0

**Comment by marvieqa** [comment]
Thanks @paulhchoi! The experience is a lot better!
I ran into some small things:
1. The links found after the H3s (ex. Find Properties, Get Access and so on...) are not recognized as links when using a screen reader. It is announced incorrectly as buttons.
2. When user selects "Learn How" button, we scroll to the content which is a visual cue to start reading. But this doesn't translate to screen reader users. We should manage the focus to go to the first `<h2>` so SR announces it immediately after scroll as a cue for them that something has changed. If this is out of scope, let me know I'll write up a separate tix.

**Comment by paulhchoi** [comment]
@marvieqa made the updates to the `The links found after the H3s` with some notes:
- the `tab-index="0"` just comes prepackaged with the `@next-ui/buttons`, so I left it in as it wasn't effecting my screenreader during testing. There's also a ticket open to refactor all buttons on the site, so this fix may come for free.

- I added the appropriate id to the `<h2 />` so the screenreader drops to it when the `Learn How` button is clicked, but on testing, the screenreader wasn't reading the h2 text immediately. Opening another ticket would work.

@thansidwell I added the links to the copy of a couple of the sections and also added the default blue/darkblue/purple styling.

**Review by thansidwell — COMMENTED** [review]
Just two more minor issues. Here's a video showing them.  https://share.cleanshot.com/1Wy92rVs

- The scroll down button sends you to far. Shift the ID from the Heading to the section div/row. 
- The buttons on the rest of the page are still too large with their padding. Make them match the "Learn How" button.

**Comment by paulhchoi** [comment]
@thansidwell regarding the scroll down button, @marvieqa requested we link the `<h2 />` anchor for screen-reader purposes. I'll revert back to adding the anchor id to the section itself, and @marvieqa, please add this as an acceptance criteria to the ticket you're opening — to ensure that the scroll action doesn't cut off the graphic on desktop viewports. Thanks!

**Comment by thansidwell** [comment]
@paulhchoi Interesting. The "Learn How" button is there to urge people to scroll down below the fold to read the other content. I'm not sure visually impaired people with screen readers behave the same way. @marvieqa, could we test what actually happens for screen readers when the ID is put on the section vs the heading? If it still need to be on the heading we can add a negative margin to make sure it doesn't go to far.

**Comment by thansidwell** [comment]
@paulhchoi I also notice that "This tool can empower..." disappears on mobile. https://share.cleanshot.com/kx293byC

**Comment by marvieqa** [comment]
> @paulhchoi Interesting. The "Learn How" button is there to urge people to scroll down below the fold to read the other content. I'm not sure visually impaired people with screen readers behave the same way. @marvieqa, could we test what actually happens for screen readers when the ID is put on the section vs the heading? If it still need to be on the heading we can add a negative margin to make sure it doesn't go to far.

@thansidwell I checked both, when @paulhchoi had it on the section and when the ID is on `<h2>`. But like Paul's experience, the screen reader did not announce it for me as well, so I'm making a separate ticket for it and do a bit of research, make recommendations from there ;).

When a screen reader user reaches "Learn How" button and activates it, there should be some form of announcement of what has changed, what did the button do. Otherwise, they will not know what has occurred, that we actually scrolled down to encourage further reading - so they are left confused. The easiest way to do this is to programmatically manage the focus to fall on something that a screen reader can announce. If the focus goes to the next heading, then they would be prompted that there are further contents to read in the page. This gives them an equitable experience as sighted users who get visual cue from the scroll action.

**Comment by paulhchoi** [comment]
@thansidwell 
> Right now the navigation scrolls away, but it should be fixed, just like on the About and Take Action pages.
> I also noticed that "This tool can empower..." disappears on mobile.

both are corrected

**Comment by paulhchoi** [comment]
@thansidwell bump for a review of this please?

**Comment by thansidwell** [comment]
Looks great! Pass.

**Comment by paulhchoi** [comment]
> Looks great! Pass.

Thanks! Now requesting a review from either @thansidwell or @nlebovits

**Review by marvieqa — APPROVED** [review]
Approved by Than! good to go!

**Pull request merged**